yoti_test.go

```diff
@@ -92,10 +94,10 @@ func TestYotiClient_HttpFailure_ReturnsProfileNotFound(t *testing.T) {
 		},
 	}
 
-	_, _, errorStrings := client.getActivityDetails(encryptedToken)
+	_, _, err := client.getActivityDetails(encryptedToken)
 
-	assert.Assert(t, len(errorStrings) > 0)
-	assert.Check(t, strings.HasPrefix(errorStrings[0], "Profile Not Found"))
+	assert.Check(t, err != nil)
+	assert.Check(t, strings.HasPrefix(err.Error(), "Profile Not Found"))
```

Comment:
In fact in all places where you are checking errors in tests it would be good to assert the temporaryness of them.